package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "react-avatar-edit",
-  "version": "0.8.3",
+  "name": "@impargo/react-avatar-edit",
+  "version": "1.0",
```

Comment:
I have a problem when trying to install, it complains because the version is 1.0 and it should be 1.0.0